app/views/user_mentee_applications/show.html.erb

```diff
@@ -79,13 +79,12 @@
       <label class='label label-text font-semibold'>GitHub URL:</label>
       <p><%= @user_mentee_application.github_url %></p>
     </div>
-  <% end %>
-
-  <%= render UserMenteeApplication::SectionComponent.new(header_text: "Job Hunting Information") do %>
```

Comment:
Section header was redundant, since the github/linkedin was also job hunting information.